include/rocksdb/cache.h

```diff
@@ -327,6 +327,7 @@ struct HyperClockCacheOptions : public ShardedCacheOptions {
   // to estimate toward the lower side than the higher side.
   size_t estimated_entry_charge;
 
+  HyperClockCacheOptions() {}
```

Comment:
It looks like there is no default value for `estimated_entry_charge`.

Here is an example of an existing C binding for an options creation API that requires a parameter to be set - https://github.com/facebook/rocksdb/blob/b7fbcefda89867dbf4919c793b18eb50dc5d8f51/include/rocksdb/c.h#L265

Can we do something similar for `rocksdb_hyper_clock_cache_options_create()` to avoid accidental use of uninitialized `estimated_entry_charge`?

Reply:
Is the default `0` for `capacity` an valid option?

Reply:
I made both capacity and estimated_entry_charge required opts.